spec/models/info_request_spec.rb

```diff
@@ -1209,6 +1209,15 @@
           with_message( "Unknown key 'random' for '#{info_request.law_used}'")
       end
 
+      context 'using the non default language' do
```

Comment:
good point, although AFAIK this uses the translation file in spec/fixtures/locale so it shouldn't be affected by Transifex changes (shouldn't)